packages/plugin-ext/src/plugin/type-converters.spec.ts

```diff
@@ -287,6 +300,14 @@ describe('Type converters:', () => {
             assert.deepStrictEqual(result, shellTaskDto);
         });
 
+        it('shell task with ShellStringQuoted command should convert to task dto', () => {
+            const result: TaskDto | undefined = Converter.fromTask(shellPluginTaskWithQuotedCommand);
+
+            // then
```

Comment:
minor: unecessary.

Reply:
I did not get which part was unnecessary @vince-fugnitto , do you mean the new ''should convert to task dto with ShellStringQuoted'' is unnecessary?

Reply:
@jdmchp I believe Vince meant the comment, which doesn't really convey any content IMO.